src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue

```diff
@@ -0,0 +1,244 @@
+<!--
+  - @copyright Copyright (c) 2022 Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="breakout-rooms">
+		<div class="breakout-rooms__actions">
+			<template v-if="breakoutRoomsConfigured">
+				<NcButton v-if="breakoutRoomsStarted"
+					v-tooltip.auto="t('spreed', 'Start breakout rooms')"
+					type="secondary"
+					@click="startBreakoutRooms">
+					<template #icon>
+						<Play :size="20" />
+					</template>
+				</NcButton>
+				<NcButton v-else
+					v-tooltip.auto="t('spreed', 'Start breakout rooms')"
+					type="secondary"
+					@click="stopBreakoutRooms">
+					<template #icon>
+						<StopIcon :size="20" />
+					</template>
+				</NcButton>
+			</template>
+
+			<!-- Configuration button -->
+			<NcButton :wide="true"
+				:type="breakoutRoomsConfigured ? 'tertiary' : 'secondary'"
+				@click="openBreakoutRoomsEditor">
+				<template #icon>
+					<DotsCircle :size="20" />
+				</template>
+				<template v-if="!breakoutRoomsConfigured">
+					{{ t('spreed', 'Configure breakout rooms') }}
+				</template>
+				<template v-else>
+					{{ t('spreed', 'Re-configure breakout rooms') }}
+				</template>
+			</NcButton>
+			<NcButton v-if="breakoutRoomsConfigured"
+				v-tooltip.auto="t('spreed', 'Delete breakout rooms')"
```

Comment:
V-tooltip is borked. Should use title+arialable instead but, as per console.warning:
> You need to fill either the text or the ariaLabel props in the button component.